rest-common/src/Cardano/Rest/Parsers.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Cardano.Rest.Parsers (pWebserverConfig) where
+
+import Cardano.Rest.Types
+    ( WebserverConfig (WebserverConfig) )
+import Network.Wai.Handler.Warp
+    ( HostPreference, Port )
+import Options.Applicative
+    ( Parser )
+import Options.Applicative
+    ( auto, help, long, metavar, option, showDefault, value )
+
+pWebserverConfig :: Parser WebserverConfig
+pWebserverConfig = WebserverConfig <$> pHostPreferenceOption <*> pPortOption
+
+pHostPreferenceOption :: Parser HostPreference
+pHostPreferenceOption =
+  option auto $
+  long "listen-address" <>
+  metavar "HOST" <>
+  help
+    ("Specification of which host to the bind API server to. " <>
+     "Can be an IPv[46] address, hostname, or '*'.") <>
+  value "127.0.0.1" <> showDefault
+
+pPortOption :: Parser Port
+pPortOption =
+  option auto $
+  long "port" <>
+  metavar "INT" <>
+  help "Port used for the API server." <> value 8090 <> showDefault
```

Comment:
For the explorer, I think the default should remain as 8100.

Reply:
👍 

Any strong feelings about the default for the submit-api? AFAICT it didn't have one before.

Reply:
I think you're right in that it doesn't have a default one right now. Feel free to pick one!

Reply:
Will do. Fix pushed!